eks-distro-base/Makefile

```diff
@@ -87,3 +92,12 @@ build: local-images
 
 .PHONY: release
 release: images
+
+.PHONY: update
+update: 
+	RETURN_VALUE="$(shell ./check_update.sh $(IMAGE_REPO) $(IMAGE_NAME) $(IMAGE_TAG))"
+	if [ ${RETURN_VALUE} = "Updates required" ]; then
+		images
+	elif [ $RETURN_VALUE = "Error" ]; then
```

Comment:
Will it ever get here?

Reply:
Yes, since I am not `exit`ing in the script, just `echo`ing what's the next step.